### PR TITLE
BZ2013874: Added a note to use same Ignition config file and ISO

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6+ versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2013874)
- [Preview](https://deploy-preview-40208--osdocs.netlify.app/openshift-enterprise/latest/machine_management/user_infra/adding-bare-metal-compute-user-infra.html)
- @pamoedom  please verify.

### CODE REVIEW

File: machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
Comment (bgilbert) [inline machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: "Recommended" may not be strong enough.  Using the same ISO image is the supported flow, and AFAIK we don't test on clusters installed from mixed bootimages.

Comment (travier) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: Agree here. We should use stronger wording.

Comment (Amrita42) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: Re-phrased as :
**You must use the same Ignition config file and ISO image that you used to install a cluster to deploy all new nodes in a cluster, before or after the cluster upgrades.**
@bgilbert @travier  what do you think?

Comment (Amrita42) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc]: Added the above .

Comment (bgilbert) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc]: This isn't strictly true of the Ignition config though.  Some users need to supply a different Ignition config for every node, e.g. for configuring a static IP address, and modifying the pointer Ignition config is currently the only way to do that.

**Review by bgilbert — APPROVED** [review]
LGTM

**Review by travier — APPROVED** [review]

File: machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
Comment (bergerhoffer) [inline machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: The first time I read this sentence, I wasn't very clear to me what the "before or after the cluster upgrades" part meant. I wasn't sure if it was trying to convey that you have to use the same ISO image that you used on install, even after the cluster has been upgraded?

I'm just wondering if that part could be removed, since you do say at the end of the note "You can add the nodes before or after the updates."

Or if you do want to keep that part in the first sentence, would something like this mean the right thing: "... all new nodes in a cluster, even after the cluster has been upgraded." or something similar?

File: machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
Comment (bergerhoffer) [inline machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:30]: This note is using a mix of "upgrade" and "update". Just wondering if that is correct, or if "upgrade" above should be "update"

**Review by bergerhoffer — COMMENTED** [review]
Just two questions!

File: machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
Comment (Amrita42) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:30]: Thanks I will keep it consistent with upgrade

File: machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
Comment (Amrita42) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: yes I would want to keep the first part and re-phrase it.

Comment (Amrita42) [reply machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc:29]: Actually on further insight, I have removed that bit `before or after the cluster upgrades`

**Review by bergerhoffer — COMMENTED** [review]
LGTM, let me know once you get QE ack and this is ready to merge!

**Pull request merged**
